config/model_configs/interactive_debug.yml

```diff
@@ -17,6 +17,6 @@ rad: "gray"
 run_name: "interactive_debug_run"
 start_date: "19790301"
 surface_setup: "PrescribedSurface"
-t_end: "20days"
+t_end: "200secs"
```

Comment:
can we change this back to 20days when it's ready?

Reply:
oops yes, I didn't mean to include this